weblate/settings_docker.py

```diff
@@ -999,6 +999,7 @@
             rf"{URL_PREFIX}/legal/(.*)$",  # Optional for legal app
         ),
     )
+    LOGIN_REQUIRE_URLS_EXCEPTIONS.extend(get_env_list("WEBLATE_EXTEND_LOGIN_REQUIRED_URLS_EXCEPTIONS"))
```

Comment:
Maybe better to use existing code to modify lists?

```suggestion
    modify_env_list(LOGIN_REQUIRE_URLS_EXCEPTIONS, "LOGIN_REQUIRE_URLS_EXCEPTIONS")
```

Also see https://docs.weblate.org/en/latest/admin/install/docker.html#changing-enabled-apps-checks-addons-or-autofixes